src/EventListener/DeserializeListener.php

```
@@ -54,6 +55,7 @@ public function onKernelRequest(GetResponseEvent $event)
             return;
         }
 
+        $format = $this->getFormat($request);
```

Because `getFormat` sometimes need to throw an exception as early as possible.